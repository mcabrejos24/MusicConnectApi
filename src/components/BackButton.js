export default function BackButton() {
    return (
        <div className="btn-container">
            <a href={ "/" }>
                <span className="back-btn">BACK</span>
            </a>
        </div>
    )
}
