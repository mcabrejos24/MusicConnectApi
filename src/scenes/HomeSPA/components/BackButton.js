import './styles/back-button.scss';
import { NavLink } from 'react-router-dom';

export default function BackButton() {

    return (
        <div className="btn-container">
            <NavLink to={ "/" }>
                <button className="back-btn">BACK</button>
            </NavLink>
        </div>
    )
}
