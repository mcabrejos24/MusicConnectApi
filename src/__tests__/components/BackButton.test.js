import React from "react";
import { shallow } from "enzyme";
import BackButton from '../../components/BackButton';
import toJson from "enzyme-to-json";

describe("renders BackButton", () => {
    let page;
    beforeEach(() => {
        page = shallow(<BackButton />);
    });
    it("BackButton loads", () => {
        page;
    });
    it("props class name matches", () => {
        expect(page.props().className).toEqual("btn-container");
    });
    it("matches snapshot", () => {
        expect(toJson(page)).toMatchSnapshot();
    });
});
