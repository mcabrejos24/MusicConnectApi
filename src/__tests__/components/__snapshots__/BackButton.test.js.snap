// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders BackButton matches snapshot 1`] = `
<div
  className="btn-container"
>
  <a
    href="/"
  >
    <span
      className="back-btn"
    >
      BACK
    </span>
  </a>
</div>
`;
